diagnostic.md

```diff
@@ -32,23 +39,26 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+Getters: .val() and .text()
+Setters: .val(value) and .text(value)
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
+The DOM is created by the browser upon running the source code as a way for javascript and the user to interact with it without changing the source code.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.